irohad/ametsuchi/impl/temporary_wsv_impl.cpp

```diff
@@ -38,10 +39,12 @@ namespace iroha {
     }
 
     bool TemporaryWsvImpl::apply(
-        const model::Transaction &transaction,
-        std::function<bool(const model::Transaction &, WsvQuery &)>
-            apply_function) {
-      const auto &tx_creator = transaction.creator_account_id;
+        const shared_model::interface::Transaction &tx,
+        std::function<bool(const shared_model::interface::Transaction &,
+                           WsvQuery &)> apply_function) {
+      auto transaction = tx.makeOldModel();
```

Comment:
This is a leak

Reply:
Also, why don't use shared_model::transaction for the work?

Reply:
Also, `transaction` very confused because `transaction_` in this scope.

Reply:
Because we used transaction's commands, and we don't have shared_model executors yet